client/my-sites/email/email-providers-comparison/price/professional-email.tsx

```diff
@@ -34,26 +35,33 @@ const ProfessionalEmailPrice = ( {
 	const productSlug = getTitanProductSlug( intervalLength );
 	const product = useSelector( ( state ) => getProductBySlug( state, productSlug ) );
 
+	if ( ! domain ) {
+		return <></>;
+	}
+
```

Comment:
Can we return `null` instead? (we might change the return type for the component too to `JSX.Component` and remove the import type)
```suggestion
	if ( ! domain ) {
		return null;
	}

```

Reply:
Will do 👍.